content/400-pulse/600-faq.mdx

```diff
@@ -145,4 +145,28 @@ A static IP address is an IPv4 or an IPv6 address that is fixed. Unlike dynamic
 
 ## Do the static IP ranges for Accelerate and Pulse differ?
 
-No, if you have static IP enabled for both Accelerate and Pulse using the same database URL, the generated static IP ranges will be the same for both products.
\ No newline at end of file
+No, if you have static IP enabled for both Accelerate and Pulse using the same database URL, the generated static IP ranges will be the same for both products.
+
+### What happens if a Pulse-created publication is manually deleted?
+
+Manually deleting a Pulse-created publication disconnects Pulse, causing a backlog in the Write-Ahead Log (WAL). While Pulse may attempt to recreate the publication automatically, it's best to avoid manually deleting publications managed by Pulse.
```

Comment:
not a huge deal but I don't understand what a "backlog" in the Write-Ahead Log is

Reply:
can this maybe be linked to the postgres docs if it's an official concept?

Reply:
```suggestion
Manually deleting a Pulse-created publication disconnects Pulse, which can lead to a backlog in the [Write-Ahead Log (WAL)](https://www.postgresql.org/docs/current/wal-intro.html), where unprocessed changes accumulate. This can slow down recovery and replication processes and may also cause storage issues as the backlog grows. While Pulse may attempt to recreate the publication automatically, it's best to avoid manually deleting publications managed by Pulse.
```